File: app/src/main/java/guillermo/com/pugme/interactors/PugsInteractor.java
package guillermo.com.pugme.interactors;

import guillermo.com.pugme.callbacks.FetchPugsUrlsCallBack;

/**
 * Created by guillermo.rosales on 21/01/16.
 */
public interface PugsInteractor {

    void fetchPugsUrls(FetchPugsUrlsCallBack callBack);
}

File: app/src/main/java/guillermo/com/pugme/interactors/PugsInteractor.kt
package guillermo.com.pugme.interactors

import guillermo.com.pugme.callbacks.FetchPugsUrlsCallBack

/**
 * Created by guillermo.rosales on 21/01/16.
 */
interface PugsInteractor {

    fun fetchPugsUrls(callBack: FetchPugsUrlsCallBack)
}
